packages/ketcher-core/src/application/editor/operations/sgroup/SGroupAttr.ts

```diff
@@ -37,7 +37,9 @@ export class SGroupAttr extends BaseOperation {
   execute(restruct: ReStruct) {
     const struct = restruct.molecule
     const sgroupId = this.data.sgid
-    const sgroup = struct.sgroups.get(sgroupId)!
+    const sgroup = struct.sgroups.get(sgroupId)
+
+    if (!sgroup) return
```

Comment:
I propose don't use single line `if`s

Reply:
I agree, there is a bigger issue, but it looks like it may relate to the way paste tool implements following the cursor. Since we will be overhauling that behaviour - I believe we can get back to this. I will add a separate task for that and link it to this one.